cmd/grafaind/scenario_test.go

```diff
@@ -178,11 +189,51 @@ func initGenesis(t *testing.T, filename string, alice weave.Address) {
 			"cash": cash.Configuration{
 				CollectorAddress: alice,
 			},
+			"msgfee": dict{
+				"owner":     alice,
+				"fee_admin": alice,
+			},
+		},
+		"rbac": dict{
+			"roles": []dict{
+				{
+					"name":  "system.admin",
+					"owner": "seq:rbac/role/1",
+					"permissions": []string{
+						"_grafain.artifacts.delete",
+					},
+				},
+				{
+					"name":     "k8s.devops",
+					"owner":    "seq:rbac/role/1",
+					"role_ids": []int{1},
```

Comment:
This means any member of role 1 (admin) is also a member of this role? (gets those permissions)?

Nice

Reply:
yes, permissions can be inherited.